docs/basics/clusters/index.md

```diff
@@ -0,0 +1,99 @@
+# Argo CD target clusters
+
+By default an Argo CD instance can deploy applications to the [same cluster](internal/) it is installed on. However, Argo CD has the capability to connect and deploy to [external clusters](external).
```

Comment:
```suggestion
By default an Argo CD instance can deploy applications to the [same cluster](internal/) it is installed on. Additionally, Argo CD has the capability to connect and deploy to [external clusters](external).
```

The use of "however" seems in opposition and I think it's better replaced with "Additionally"